substrate/runtime/primitives/src/traits.rs

```diff
@@ -147,10 +149,86 @@ impl<A: Executable, B: Executable> Executable for (A, B) {
 	}
 }
 
+/// Abstraction around hashing
+pub trait Hashing {
```

Comment:
more idiomatic to call it `Hash` since it describes a hash function

Reply:
it's actually just moving existing code around, so rather out of scope for this pr